ush/generate_FV3LAM_wflow.py

```diff
@@ -812,6 +559,248 @@ def generate_FV3LAM_wflow(
 
     return EXPTDIR
 
+def setup_fv3_namelist(expt_config,debug):
+    """
+    Updates parameters specific to the FV3ATM namelist for the run_fcst step.
+
+    Args:
+        expt_dict (dict): The full experiment configuration dictionary
+        debug    (bool): Enable extra output for debugging
+    Returns:
+        EXPTDIR (str) : The full path of the directory where this experiment has been generated
+    """
+
+    # From here on out, going back to setting variables for everything
+    # in the flattened expt_config dictionary
+    # TODO: Reference all these variables in their respective
+    # dictionaries, instead.
+    import_vars(dictionary=flatten_dict(expt_config))
+    export_vars(source_dict=flatten_dict(expt_config))
```

Comment:
I made some similar clean up changes in my PR. I find that it's safer to limit the sections that you export here and you can avoid flattening for this section for the most part with:

```
export_vars(source_dict=expt_config["global"])
```